src/aboutkuzya.h

```diff
@@ -0,0 +1,37 @@
+#ifndef ABOUTKUZYA_H
+#define ABOUTKUZYA_H
+
+#include <QDialog>
+#include <QString>
+#include <QFile>
+#include <QTextStream>
+#include <QVersionNumber>
+#include <QDate>
+
+namespace Ui {
+  class AboutKuzya;
+}
+
+class AboutKuzya : public QDialog
+{
+  Q_OBJECT
+
+public:
+  AboutKuzya(QVersionNumber verKuzia, QDate buildDate, QString fileName, QWidget *parent = 0);
+  ~AboutKuzya();
+
+private:
+  Ui::AboutKuzya *ui;
+  QString *mFileName;
+  QFile *mFile;
```

Comment:
Again, you do not need all this pointers. You could create string, stream and file objects locally in your method.